### PR TITLE
ENH: Translating Mesh3 C++ example into Python

### PR DESCRIPTION
**Pull request by dzenanz**
blocker: identifiers beginning with a digit are not supported
```
  File "./Code.py", line 29
    CellTraits = itk.itkCellTraitsInfo.2FFULLULLULLPF2MCULLPF2
                                       ^
SyntaxError: invalid syntax
```

### CODE REVIEW

**Comment by thewtex** [comment]
```
  File "./Code.py", line 29
    CellTraits = itk.itkCellTraitsInfo.2FFULLULLULLPF2MCULLPF2
                                       ^
SyntaxError: invalid syntax
```

@fbudin69500 I seem to recall we discussed this issue in the past. Did we end up changing the name mangling so it did not start with a number?

**Comment by thewtex** [comment]
@dzenanz could this please be rebased on `master` and updated to use `itk.meshread`, etc?

**Comment by thewtex** [comment]
Rebased and updated. We need to add LineCell wrapping.

**Comment by PranjalSahu** [comment]
@dzenanz can you please allow edits by maintainers. I can push my changes to your branch then.

**Comment by dzenanz** [comment]
> allow edits by maintainers

I already do. I now added you as a maintainer to the repository.

**Comment by dzenanz** [comment]
@PranjalSahu please rename the PR once it is ready. As I cannot review my own PR, at least you should review it.

**Comment by dzenanz** [comment]
LGTM

File: src/Core/Mesh/AccessDataInCells/Code.cxx
Comment (dzenanz) [inline src/Core/Mesh/AccessDataInCells/Code.cxx:24]: This could should be updated based on the source:
https://github.com/InsightSoftwareConsortium/ITK/blob/047fa6934ac32711d1d7b84fa6eb81c72db01960/Examples/DataRepresentation/Mesh/Mesh3.cxx
In particular, `using` instead of `typedef`, but many other style changes too. A lot has changed over the last 4 years, since this PR was started 😄

File: src/Core/Mesh/AccessDataInCells/Code.cxx
Comment (tbirdso) [inline src/Core/Mesh/AccessDataInCells/Code.cxx:30]: `auto mesh = MeshType::New();`

**Review by tbirdso — CHANGES_REQUESTED** [review]
Looks great! Similar to @dzenanz 's review, a couple of requests for style updates.

**Comment by PranjalSahu** [comment]
ok I will do the changes.

**Review by tbirdso — APPROVED** [review]
Thanks @PranjalSahu !

**Review by thewtex — APPROVED** [review]
Awesome! :1st_place_medal: :champagne: :fireworks:

**Pull request merged**
